samples/sensor/6dof_motion_drdy/boards/nrf52dk_nrf52832_i2c.overlay

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2024, TDK Invensense
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*
+ * Get a node identifier for 6-axis IMU sensor.
+ */
+/ {
+	aliases {
+		6dof-motion-drdy0 = &icm42670;
+	};
+};
+
+/*
+ * Example configuration of a ICM42670 device on i2c0 compatible with an Arduino I2C bus.
```

Comment:
Is there an arduino shield available with this sensor?

Reply:
No, there is no Arduino shield available for now.